photonai_neuro/registered

```diff
@@ -0,0 +1 @@
+2020-08-25 18:17:23.398837
```

Comment:
do we have to pay attention by pushing this file? already thought about an registration every time photonai_neuro __init__.py is called..is it possible?

Reply:
yes you're right. We should put it in the gitignore.